src/components/authoring/authoring-app.tsx

```diff
@@ -19,6 +19,10 @@ export const DEFAULT_GLOSSARY: IGlossary = {
   definitions: []
 };
 
+interface IProps {
```

Comment:
Why do we need to pass the accessToken as a prop, it doesn't seem to be used in the component.

Reply:
Right, it's not used yet, but it will be soon. I pushed that to differentiate the result of authorization or lack of it.